blueprints/component-test/index.js

```diff
@@ -1,3 +1,61 @@
 module.exports = {
-  description: 'Generates a component unit test.'
+  description: 'Generates a component test.',
+
+  availableOptions: [
+    {
+      name: 'integration',
+      type: Boolean,
+      default: false,
+      aliases: ['i']
+    }
+  ],
+
+  locals: function(options) {
+    var additionalImports, prefix, testOpts, defaultTest;
+    if (options.integration) {
+      additionalImports = "import hbs from 'htmlbars-inline-precompile';";
```

Comment:
Hmm, without having the `htmlbars-inline-precompile` dependency installed, a newly created integration test will fail... Should it be added to the dependencies in `package.json`?